activerecord/CHANGELOG.md

```diff
@@ -543,4 +543,9 @@
 
     *Yves Senn*
 
+*   ':time' option added for '#touch'
```

Comment:
You can wrap these in backticks instead of single quotes, so that it highlights properly as markdown. You can see [the formatted file in github](https://github.com/hjoo/rails/blob/time_option/activerecord/CHANGELOG.md).